dev/io.openliberty.microprofile.telemetry.1.0.internal.jaeger_fat/fat/src/io/openliberty/microprofile/telemetry/internal/utils/otelCollector/OtelCollectorContainer.java

```diff
@@ -0,0 +1,77 @@
+/*******************************************************************************
+ * Copyright (c) 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.microprofile.telemetry.internal.utils.otelCollector;
+
+import java.io.File;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+import org.testcontainers.utility.DockerImageName;
+import org.testcontainers.utility.ImageNameSubstitutor;
+
+/**
+ * A container for the otelCollector trace server
+ * <p>
+ * Usage:
+ *
+ * <pre>
+ * {@code @ClassName}
+ * public static otelCollectorContainer otelCollectorContainer = new OtelCollectorContainer("exporter").withLogConsumer(new SimpleLogConsumer(MyTest.class, "otelCol"));
+ * </pre>
```

Comment:
You've updated this documentation, but then you also changed the constructor signature so it needs updating again :(